src/styles/main.scss

```diff
@@ -1,3 +1,69 @@
 body {
   margin: 0;
+  padding-top: 10px;
+}
+
+$transition-time: 0.6s;
+$wite: #eee;
+$first-color: #000;
+$hover-color: #ffb6c1;
+
+$days: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
+.calendar {
+  width: 760px;
+  padding: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  @for $i from 28 through 31 {
+    &--month--#{$i} > :nth-child(n + #{$i + 1}) {
+      display: none;
+    }
+  }
+
+  @each $day, $order in $days {
+    &--start-day--#{$day} > :first-child {
+      margin-left: $order * (100px + 1px);
+    }
+  }
+
+  &__cell {
+    display: flex;
+    margin-top: 1px;
+    box-sizing: border-box;
+    width: 100px;
```

Comment:
It's better to move dimensions, like width and height, to vars too, as well as gap between cells